config/update.go

```diff
@@ -0,0 +1,247 @@
+package config
+
+import (
+	"encoding/json"
+	"errors"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"regexp"
+	"runtime"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var appName = "multi_repo_extractor"
+var apiURL = "https://api.github.com/repos/codersrank-org/multi_repo_extractor/releases/latest"
+var currentVersion = version{
+	Major: 1,
+	Minor: 2,
+	Patch: 0,
+}
+
+// CheckUpdates checks github to see if there is a new version and if there is one, downloads it.
+func CheckUpdates() {
```

Comment:
After downloading the binary it is not executable. I got `permission denied`. After `chmod +x ./multi_repo_extractor` it worked.
Change the permission of the new file in the code in case of Linux and osx.

Reply:
Once it exists it works fine but the first time it might be a problem.